### ISSUE TITLE
OpenGL causes graphical and audio issues in Alice in Wonderland (USA) 

### ISSUE THREAD

**Issue by Kurayami7** [Arisotura/melonDS#1069]
When using OpenGL the music on the title screen turns into a crackling mess. Software sounds fine, OpenGL with JIT enabled sounds fine, but OpenGL by itself is broken. It also creates a black line in the middle of the screen that isn't there with software.

![Capture2](https://user-images.githubusercontent.com/77198254/115949907-6e68e400-a51b-11eb-9dc5-39e9c44e83eb.PNG)

**Comment by Brankale** [comment]
The audio is crackling because you are not reaching full speed. Enabling JIT solves the problem because it helps to improve performance. For the black line, I would say it's a screen gap of 1px since it's perfectly in the middle and you can disable it in the options of the emulator, but if you say it's not there with the software renderer maybe it's another problem.

**Comment by poudink** [comment]
yeah pretty sure the audio is a performance problem there, nothing to do with opengl. it doesn't happen on software because software at 1x is most of the time faster than opengl.

**Comment by iamthelaw2041** [comment]
This is still an issue. With OpenGL + no JIT the game runs at 57/60 speed and the music breaks. My computer should be more than capable of running this game full speed. 

![speeccy](https://user-images.githubusercontent.com/94656803/144640763-194d25f7-7917-473c-8895-24a6272d2f8f.PNG)

**Comment by nadiaholmquist** [comment]
This is not a bug. Some games can be demanding, especially without the JIT.
